livepeer_cli error when getting stats from older version of livepeer

**Describe the bug**
When launching livepeer_cli version 0.5.35 with an older version of livepeer, the following error occurs while trying to get statistics (that don't exist in older version) from livepeer at runtime.

```
panic: interface conversion: interface {} is nil, not map[string]interface {}

goroutine 1 [running]:
main.(*wizard).getBroadcasterPrices(0xc0003a9b00)
        /home/runner/work/go-livepeer/go-livepeer/cmd/livepeer_cli/wizard_stats.go:499 +0x4c6
main.(*wizard).orchestratorStats(0xc0003a9b00)
        /home/runner/work/go-livepeer/go-livepeer/cmd/livepeer_cli/wizard_stats.go:213 +0x6b
main.(*wizard).stats(0xc0003a9b00, 0x1)
        /home/runner/work/go-livepeer/go-livepeer/cmd/livepeer_cli/wizard_stats.go:90 +0xcb5
main.(*wizard).run(0xc0003a9b00)
        /home/runner/work/go-livepeer/go-livepeer/cmd/livepeer_cli/livepeer_cli.go:157 +0x2c5
main.main.func1(0xc0002ccc60)
        /home/runner/work/go-livepeer/go-livepeer/cmd/livepeer_cli/livepeer_cli.go:57 +0x4f8
github.com/urfave/cli.HandleAction({0x2bab80, 0x43de70}, 0xc0005a6c40)
        /home/runner/go/pkg/mod/github.com/urfave/cli@v1.22.2/app.go:523 +0xa8
github.com/urfave/cli.(*App).Run(0xc0005a6c40, {0xc0000322e0, 0x1, 0x1})
        /home/runner/go/pkg/mod/github.com/urfave/cli@v1.22.2/app.go:285 +0x734
main.main()
        /home/runner/work/go-livepeer/go-livepeer/cmd/livepeer_cli/livepeer_cli.go:63 +0x34e
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install Livepeer release 0.5.34 package binaries 
2. Update only `livepeer_cli` to release 0.5.35
3. Run `livepeer_cli`
4. See error

**Expected behavior**
- livepeer_cli should launch without error, it should display `Base price for broadcasters` as a blank string.
- Option 20 "Set price per broadcaster" might also need to be disabled, as the feature did not exist in older version and would not work. Test for any backwards compatible issues.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04

--- a/CHANGELOG_PENDING.md
+++ b/CHANGELOG_PENDING.md
@@ -17,6 +17,7 @@
 - \#2686 Control non-stream specific scene classification with command line args
 
 ### Bug Fixes 🐞
+- \#2697 Fix backwards compatibility of livepeer_cli with prior livepeer version
 
 #### CLI
 

--- a/cmd/livepeer_cli/wizard_stats.go
+++ b/cmd/livepeer_cli/wizard_stats.go
@@ -496,9 +496,12 @@ func (w *wizard) getBroadcasterPrices() (string, error) {
 	}
 
 	prices := new(bytes.Buffer)
-	for b, p := range status["BroadcasterPrices"].(map[string]interface{}) {
-		if b != "default" {
-			fmt.Fprintf(prices, "%s: %s per pixel\n", b, p)
+
+	if broadcasterPrices, ok := status["BroadcasterPrices"]; ok {
+		for b, p := range broadcasterPrices.(map[string]interface{}) {
+			if b != "default" {
+				fmt.Fprintf(prices, "%s: %s per pixel\n", b, p)
+			}
 		}
 	}
 

